blocks/footer/footer.js

```diff
@@ -10,7 +10,7 @@ export default async function decorate(block) {
   block.textContent = '';
 
   // load footer fragment
-  const footerPath = footerMeta.footer || '/footer';
+  const footerPath = footerMeta ? new URL(footerMeta, window.location).pathname : '/footer';
```

Comment:
This also fixes the bug (https://github.com/adobe/aem-boilerplate/pull/264) with `footerMeta` now being a string and not an object so `footerMeta.footer` can never be valid.